menas/src/main/scala/za/co/absa/enceladus/menas/services/SchemaRegistryService.scala

```diff
@@ -69,8 +114,34 @@ class SchemaRegistryService @Autowired()() {
    */
   def loadSchemaByUrl(remoteUrl: String): SchemaResponse = {
     try {
+      if (keyManagerFactory.isEmpty) {
+        logger.warn("keyManagerFactory is not, secure schema registry integration may not function properly. " +
+          s"Set ${SecureConfig.Keys.javaxNetSslKeyStore} and ${SecureConfig.Keys.javaxNetSslKeyStorePassword} to fix the issue.")
+      }
+      if (trustManagerFactory.isEmpty) {
+        logger.warn("trustManagerFactory is not, secure schema registry integration may not function properly. " +
```

Comment:
is not defined? 
Just `is not` seems lacking a word

Reply:
yes, definitely, thank you!